docs/config.md

```diff
@@ -448,6 +448,11 @@ branches. Since the local branch isn't created, the remote branch will be
 deleted if you push the branch with `jj git push --branch` or `jj git push
 --all`.
 
+### Prefix for generated branches on push
+
+`jj git push --change` generates branch names with a prefix of "push-" by
+default. You can pick a different prefix by setting `push.branch-prefix`.
```

Comment:
Add an example?

    push.branch-prefix="martinvonz/push-"

Reply:
Our you could do `your_initials/push-`.

Reply:
Good idea. Done.